### PR TITLE
windows - git populate scripts

### PR DESCRIPTION
**Pull request by GermaVinsmoke**
Solution for #739

Signed-off-by: GermaVinsmoke <vaibhav1180@gmail.com>

### CODE REVIEW

**Review by hellais — APPROVED** [review]
LGTM

File: package.json [hunk 1]
```diff
@@ -78,19 +78,19 @@
...
-    "export": ". ./scripts/populate-git-env.sh && next export",
+    "build": "next build",
+    "export": "next export",
     "lint": "next lint",
     "test:e2e": "yarn build && start-server-and-test start http://localhost:3100 cypress:run",
     "cypress:run": "cypress run",
     "test": "yarn run test:e2e",
     "script:build-translations": "node ./scripts/build-translations.js",
     "script:extract-i18n-keys": "yarn build && node ./scripts/extract-react-intl-keys.js",
-    "script:populate-git-env:print": ". ./scripts/populate-git-env.sh && echo 'SHA: '${GIT_COMMIT_SHA} && echo 'SHA (short): '${GIT_COMMIT_SHA_SHORT} && echo 'REF (current branch/tag): '${GIT_COMMIT_REF} && echo 'TAGS: '${GIT_COMMIT_TAGS}",
```
Comment (hellais) [inline package.json:92]: Before merging you should delete the script if it's no longer being used.

Comment (GermaVinsmoke) [reply package.json:92]: Removed

Comment (majakomel) [reply package.json:92]: We can also remove `/scripts/populate-git-env.sh` if it's not being used anymore.

Comment (GermaVinsmoke) [reply package.json:92]: checked - not used anywhere.
So, removed it

**Pull request merged**